feat: hide tos checkbox optional (#13823)
shopware/platform: RELEASE_INFO-6.7.md

## Features

> ### TOS checkbox position update in 6.8

Reviewer: Must also be a UPGRADE-6.8.md entry, right?

Reviewer 2: this should only go the upgrade guide for 6.8, as this has no effect in the current 6.7 major line

> Starting with version 6.8, the Terms of Service (TOS) checkbox will be relocated to the bottom of the order confirmation page, where it will be hidden by default and replaced with a descriptive label, while its visibility can be controlled using the new configuration option core.cart.showTosCheckbox.

Reviewer 3: ```suggestion
Starting with version 6.8, the Terms of Service (TOS) will be relocated to the bottom of the order confirmation page. The checkbox will be hidden by default due to not being necessary and replaced with a descriptive label, while its visibility can be controlled using the new configuration option core.cart.showTosCheckbox.
```
